tests/check/test_stream.c

```diff
@@ -120,6 +120,8 @@ get_type_char(const uint8_t *data, size_t data_size, SignedVideoCodec codec)
         type = 'Z';
       else if (bu.is_golden_sei)
         type = 'G';
+      else if (bu.uuid_type == UUID_TYPE_ONVIF_MEDIA_SIGNING)
+        type = 'O';
```

Comment:
I'm missing an update of `test_stream_item_create_from_type()`.
If I generate a test stream like "IPPIOPPIOP" I'd expect a correct linked list of items.